### DISCUSSION TITLE
Will the sidebar support collapsable function?

### DISCUSSION THREAD

**Discussion by jevantang** [vuepress/core#279] [Q&A] [answered]
I am going to upgrade v1 to v2, but I have been waiting for the collapsable function of the sidebar. I wonder if the default theme of v2 will still be supported?

My scenario is like this. My product is also open source software. The documentation involves more than 50 data tables and more than 100 API interfaces. With such a large number, the sidebar is very long, so I perform data tables and API documents.  In this way, the sidebar is very refreshing, and there are not too many lists, which affects the look and feel.

**Comment by Mister-Hope** [comment, marked as answer]
Emm, found another one.😎Added
